packages/minter-contracts/ligo/src/fixed_price_sale/README.md

```diff
@@ -0,0 +1,265 @@
+contains the definitions of the `init_sale_param` and `init_sale_param_tez`
+records.
+
+### %sell
+
+While the purpose of this entrypoint should be readily obvious, it still is
+necessary to point out some important details whilst using it. Firstly, if there
+exists an administrator of the given fixed price sale contract, that
+administrator can pause the contract. If such an occurrence happens, then a
+potential seller cannot initiate a sale. This entrypoint invocation will fail
+with a message satisfying the FA2 specification.
+
+Provided an un-paused fixed price sale contract, this entrypoint's invocation
+will initiate a transfer of the seller's NFT token to the marketplace. It is
+CRITICAL that the seller add the marketplace contract as another operator of
+their NFT token before initiating the sale. Find some accompanying tests here
+(they contain instructive examples):
+
+* [nft_market.test.ts](../../../test/nft_market.test.ts) lines 91, 136
+* [nft_market_fa2.test.ts](../../../test/nft_market_fa2.test.ts) lines 143, 146,
+  192, 195
+* [nft_market_fa2_with_admin.test.ts](../../../test/nft_market_fa2_with_admin.test.ts)
+  lines 164, 172, 257
+* [nft_market_with_admin.test.ts](../../../test/nft_market_with_admin.test.ts)
+  lines 97, 176.
+
+The definition of the function, `addOperator`, is here:
+
+* [fa2_interface.ts](../../../src/fa2-interface.ts) line 46.
+
+
+These above steps outline apply irrespective of which version of the fixed price
+sale contract is in use. The primary difference is the record type of the
+variant associated with this entrypoint. For [`SALE_FA2`](#sale-fa2-section) we
+have the record type `init_sale_param` we have:
+
+
+``` ocaml
+type init_sale_param =
+[@layout:comb]
+{
+  sale_price: nat;
+  sale_tokens_param: sale_tokens_param;
+}
+```
+
+This record contains the sale price of the NFT in positive integer units
+corresponding to the FA2 token used to price the NFT along with
+`sale_tokens_param` field described earlier in this document (the section
+[Storage](#storage-section)). We have something similar for
+[`SALE_TEZ`](#sale-tez-section):
+
+``` ocaml
+type init_sale_param_tez =
+[@layout:comb]
+{
+  sale_price: tez;
+  sale_token_param_tez: sale_token_param_tez;
+}
+```
+
+This record, like the one above, contains the sale price in tez (instead of some
+FA2 token) and the `sale_token_param_tez`, again described earlier in this
+document (the section [Storage](#torage-section)).
+
+### %buy
+
+Like in the previous section, an administrator can prevent this entrypoint's
+invocation by pausing the contract. Nonetheless, under normal operation, this
+entrypoint will allow a potential buyer to claim the NFT for sale provided they
+have funds. If they do not have the funds, they will not lay claim to the NFT
+(it will remain under the ownership of the seller). If someone else has bought
+the NFT or if the seller has cancelled the sale, the entrypoint will emit an
+error message of "NO_SALE".
+
```

Comment:
Should this section explain what would happen when `%buy` entry point is called?
Withdrawing buyer's money token; transferring NFTs to the buyer etc.
Should the buyer set market contract as an operator of its money token before a buy?